LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
are you sure MIT is suitable for this content? Shouldn't it be some kind of Creative Commons License?

Reply:
fixed 7e0cb09 - Switched to CC-BY-4.0